duo_admin/plugin.spec.yaml

```diff
@@ -429,7 +457,80 @@ actions:
+        title: Maxtime
+        type: integer
+        description: Maximum time in UNIX timestamp milliseconds. Must be 13 or more digits in length and greater than mintime. To use current time leave this parameter empty
+        required: false
+        example: 1611069760000
+      applications:
+        title: Applications
+        type: '[]string'
+        description: List of application IDs to filter on
+        required: false
+        example: ["DIV9C5V7T6L02DRWL4RU"]
+      users:
+        title: Users
+        type: '[]string'
+        description: List of user IDs to filter on
+        required: false
+        example: ["DUW2DKA44RFYECTU8R1O"]
+      event_types:
+        title: Event Types
+        type: '[]string'
+        description: List of event types(authentication, enrollment) to filter on, to include all leave this parameter empty
+        required: false
+        example: ["authentication"]
+      factors:
+        title: Factors
+        type: '[]string'
+        description: List of factors or methods used for an authentication attempt(duo_push, phone_call, u2f_token, hardware_token, bypass_code, sms_passcode, duo_mobile_passcode, yubikey_code, passcode, digipass_go_7_token, not_available, sms_refresh, remembered_device, trusted_network) to filter on, to include all leave this parameter empty
+        required: false
+        example: ["duo_push", "sms_passcode"]
+      groups:
+        title: Groups
+        type: '[]string'
+        description: List of group IDs to filter on
+        required: false
+        example: ["DG67EON0I1QA2ZDUF32M"]
+      phone_numbers:
+        title: Phone Numbers
+        type: '[]string'
+        description: List of phone numbers to filter on
+        required: false
+        example: ["+11111111111"]
+      reasons:
+        title: Reasons
+        type: '[]string'
+        description: List of reasons associated with an authentication attempt(user_marked_fraud, deny_unenrolled_user, error, locked_out, user_disabled, user_cancelled, invalid_passcode, no_response, no_keys_pressed, call_timed_out, location_restricted, factor_restricted, platform_restricted, version_restricted, rooted_device, no_screen_lock, touch_id_disabled, no_disk_encryption, anonymous_ip, out_of_date, denied_by_policy, software_restricted, no_duo_certificate_present, user_provided_invalid_certificate, could_not_determine_if_endpoint_was_trusted, invalid_management_certificate_collection_state, no_referring_hostname_provided, invalid_referring_hostname_provided, no_web_referer_match, endpoint_failed_google_verification, endpoint_is_not_trusted, invalid_device, anomalous_push,  endpoint_is_not_in_management_system, no_activated_duo_mobile_account, allow_unenrolled_user, bypass_user, trusted_network, remembered_device, trusted_location, user_approved, valid_passcode, allowed_by_policy, allow_unenrolled_user_on_trusted_network, user_not_in_permitted_group) to filter on, to include all leave this parameter empty
```

Comment:
We should remove the list of reasons here and point the user to the help documentation for the plugin to see all available inputs